src/runtime/app.ts

```diff
@@ -120,6 +125,7 @@ function createNitroApp(): NitroApp {
     router,
     localCall,
     localFetch,
+    localFetchWithNormalizedHeaders,
```

Comment:
We should directly apply patch to `localFetch` if for (any) reason you think we need an escape hatch to disable the normalization of cookie header, it can be a custom fetch option however i think we should not need it

Reply:
We are no longer returning the full Response, but a subset of it that contains body, headers, status and statusText. 
This matters for $fetch usage. 
Should we try to return everything and change localFetch itself.

Reply:
Actually nvm I think it should work

Reply:
We can make a internal `_localFetch` variable only being passed to `createFetch` but i guess if standard is to split cookies, it is also nice to be applied to subsequent local `$fetch` calls..

What is more than body, headers, status, statusText available in a Response object?